awswrangler/data_quality/_create.py

```diff
@@ -0,0 +1,196 @@
+"""AWS Glue Data Quality Create module."""
+
+import logging
+import uuid
+from typing import Dict, List, Optional
+
+import boto3
+import pandas as pd
+
+from awswrangler import _utils, exceptions
+from awswrangler._config import apply_configs
+from awswrangler.data_quality._utils import (
+    _get_data_quality_results,
+    _start_ruleset_evaluation_run,
+    _wait_ruleset_evaluation_run,
+)
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+def _create_dqdl(
+    df_rules: pd.DataFrame,
+) -> str:
+    """Create DQDL from pandas data frame."""
+    rules = []
+    for rule_type, parameter, expression in df_rules.itertuples(index=False):
+        parameter_str = f' "{parameter}" ' if parameter else " "
+        expression_str = expression if expression else ""
+        rules.append(f"{rule_type}{parameter_str}{expression_str}")
+    return "Rules = [ " + ", ".join(rules) + " ]"
+
+
+@apply_configs
+def create_ruleset(
```

Comment:
We should enable the user to update an existing ruleset with a DF. Question is should it be a separate API or should this one handle the case where the ruleset already exists and just update it?

Reply:
Good idea, I've added a separate `update_ruleset` for now but happy to merge those two into a single method. 

There are tiny annoying differences about update - for, example, looks like you can't update Glue database/table associated with the ruleset. You can also rename an existing ruleset so if we merge them, we'd need an additional parameter. For those reasons I lean slightly towards keeping them separate but no strong preference here.